Automattic/jetpack
# WordPress.com Block Editor: disable if blocks are disabled

Fixes #18958

#### Changes proposed in this Pull Request:

* One can use the `jetpack_gutenberg` filter to disable all Jetpack blocks at once. When you do so, you should not expect any other Jetpack block editor enhancements from being loaded on the site.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* On a site that's connected to WordPress.com, load the block editor.
* You should see `https://widgets.wp.com/wpcom-block-editor/default.editor.min.js` being loaded. You should also see the Jetpack blocks available.
* Add `add_filter( 'jetpack_gutenberg', '__return_false' );` somewhere on your site.
* Refresh the editor.
* No Jetpack blocks should appear, and the file above should not be loaded anymore.

#### Proposed changelog entry for your changes:

* WordPress.com Block Editor: disable if all Jetpack blocks are disabled via a filter.
